Cannot run the tests in continuous integration

Some tests fail both on stable and on nightly compilers which means we cannot use CI on this crate at the moment, cf #23 and particularly: https://github.com/rust-embedded/nb/runs/801586296?check_suite_focus=true

On nightly we get:
```
---- src/lib.rs -  (line 293) stdout ----
error[E0599]: no method named `resume` found for generator `[generator@src/lib.rs:15:22: 29:6 _]` in the current scope
  --> src/lib.rs:332:16
   |
40 |         blinky.resume();
   |                ^^^^^^ method not found in `[generator@src/lib.rs:15:22: 29:6 _]`

error[E0599]: no method named `resume` found for generator `[generator@src/lib.rs:31:24: 36:6 _]` in the current scope
  --> src/lib.rs:333:18
   |
41 |         loopback.resume();
   |                  ^^^^^^ method not found in `[generator@src/lib.rs:31:24: 36:6 _]`

error: aborting due to 2 previous errors
```

On stable we receive tons of errors about unstable `#[feature]` attributes.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -298,8 +298,9 @@
 //! #[macro_use(await)]
 //! extern crate nb;
 //!
-//! use std::ops::Generator;
-//! use std::pin::Pin;
+//! extern crate core;
+//! use core::ops::Generator;
+//! use core::pin::Pin;
 //!
 //! use hal::{Led, Serial, Timer};
 //!

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -299,6 +299,7 @@
 //! extern crate nb;
 //!
 //! use std::ops::Generator;
+//! use std::pin::Pin;
 //!
 //! use hal::{Led, Serial, Timer};
 //!
@@ -327,10 +328,12 @@
 //!         }
 //!     };
 //!
+//!     let blinky = Pin::new(&mut blinky);
+//!     let loopback = Pin::new(&mut loopback);
 //!     // Event loop
 //!     loop {
-//!         blinky.resume();
-//!         loopback.resume();
+//!         blinky.resume(());
+//!         loopback.resume(());
 //!         # break
 //!     }
 //! }

